.travis.yml

```diff
@@ -1,7 +1,11 @@
 language: go
 
 go:
-  - 1.7
+  - "1.7"
+  - "1.8"
+  - "1.9"
+  - "1.10"
+  - "1.11"
```

Comment:
given what the travis checks are doing, i don't think we need all these versions.  it's not actually building the product.

(which is also why it's fine that it's at 1.7 even though origin itself can't build with 1.7).

so i don't think this is really necessary right now.